### DISCUSSION TITLE
missing doc for raycasting

### DISCUSSION THREAD

**Discussion by Myziak** [enable3d/enable3d#335] [Help / Questions]
Hello,

I need to create an interactive click that returns the coordinates of the intersection with my terrain. 
![image](https://user-images.githubusercontent.com/75075069/195692991-4e206001-fc80-4491-80a0-69d65aee219e.png)

i wanted to use raycasting, but there is no documentation yet, and the example is old (throws error about getCollisionObject();)
i updated enable3d to actuall version. 

How i can get x,y,z coords of clicked terrain?

**Comment by yandeu** [comment]
If your terrain doesn't use physics, you can simply use the [three.js raycaster](https://threejs.org/docs/#api/en/core/Raycaster).
